boost-maven/boost-runtimes/runtime-openliberty/src/main/java/org/microshed/boost/runtimes/openliberty/LibertyRuntime.java

```diff
@@ -69,6 +70,7 @@ public LibertyRuntime() {
         this.projectBuildDir = null;
         this.libertyServerPath = null;
         this.mavenDepPlugin = null;
+        this.runtimeVersion = defaultRuntimeVersion;
```

Comment:
Or should we use the RELEASE qualifier instead?   Especially now that there'd be a way to stick to a given, fixed runtime version